### PR TITLE
anime-downloader 4.0.1

### PR DESCRIPTION
**Pull request by nandahkrishna**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
-----
This PR aims to update anime-downloader to version 4.0.1, and has some changes in addition to the version bump. Additional dependencies and resources have been added due to significant changes and tests have been rewritten.

### CODE REVIEW

**Review by chenrui333 — APPROVED** [review]
lgtm

File: Formula/anime-downloader.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
 class AnimeDownloader < Formula
   include Language::Python::Virtualenv
 
-  desc "Simple but powerful anime downloader and streamer"
+  desc "Download your favourite anime"
```
Comment (chenrui333) [inline Formula/anime-downloader.rb]: Not quite sure why we need to update the `desc` but the old desc seems good.

Comment (nandahkrishna) [reply Formula/anime-downloader.rb]: Ah, just updated it to ensure consistency with the description on PyPI.

Comment (chenrui333) [reply Formula/anime-downloader.rb]: Sounds cool!

**Comment by chenrui333** [comment]
Thanks @nandahkrishna !!

**Pull request closed without merging**